Safety/AttitudeManager/Inc/attitudeStateClasses.hpp

```diff
@@ -10,11 +10,21 @@
 #include "OutputMixing.hpp"
 #include "PID.hpp"
 #include "SendInstructionsToSafety.hpp"
+#include "PWM.hpp"
 
 /***********************************************************************************************************************
  * Code
  **********************************************************************************************************************/
 
+class pwmSetup : public attitudeState
+{
+    public:
+        void enter(attitudeManager* attitudeMgr) {(void) attitudeMgr;}
+        void execute(attitudeManager* attitudeMgr);
+        void exit(attitudeManager* attitudeMgr);
+        static attitudeState& getInstance();
+}
+
```

Comment:
The pwmSetup state is used to call upon functions for setup that are only called once, like setting up the PWM channels. This state only runs once, as after running the outputmixing state it cycles back to the fetchInstructions state.

Reply:
so everything in pwmSetup only run once at the start of flight?

Reply:
it should only run once, since when AM is initialized its in the setup state, and after outputmixing it goes back into the fetchInstructions state instead of pwmSetup.

Reply:
yes that is correct